kafka/mockcluster.go

```diff
@@ -0,0 +1,73 @@
+package kafka
+
+import "C"
+import "unsafe"
+
+/*
+#include <stdlib.h>
+#include "select_rdkafka.h"
+#include "glue_rdkafka.h"
+*/
+import "C"
+
+type MockCluster struct {
+	handle   handle
```

Comment:
Why are using handle here and not just `*C.rd_kafka_t`?  It doesn't seem the other fields of the handle is used anywhere.